.changeset/strong-colts-dance.md

```diff
@@ -0,0 +1,28 @@
+---
+'xstate': major
+---
+
+The `each()` action creator has been added for SCXML compatibility:
+
+```js
+// ...
+context: {
+  todos: [/* ... */],
+  todo: null, // will be populated by `each` action
+  currentTodo: null // will be populated by `each` action
+}
+actions: each([
+  (context, event) => {
+    console.log({
+      todo,
+      currentTodo
+    });
+  },
+  // ... action objects
+], {
+  array: 'todos',
```

Comment:
we could potentially handle other iterables as well, maybe worth renaming this to a `collection`? OTOH not sure if we actually want to handle non-arrays from the start and another name would definitely indicate that this can handle more than arrays 🤔

Reply:
The name `array` is meant to match SCXML, as the only intention for the action creator is to support SCXML. Should I revise the changeset?